lib/full.css

```diff
@@ -14,6 +14,29 @@
   color: <<FORE_PRIMARY>>;
 }
 
+.sj-search-box-0,
```

Comment:
For instance if we used to DOM tag selector instead of the class

`sj-search-box`